docs/UserGuide.md

```diff
@@ -127,16 +125,30 @@ More examples:
 * The indexes **must not be duplicated**. For example, `delete 3 3` will not work.
 </div>
 
-### Importing your timetable : `import`
-
-Imports your NUSMods timetable data into your schedule.
+### Importing timetable : `import`
 
 Format: `import url/YOUR_NUSMODS_URL`
 
-* Lesson data based on your NUSMods timetable will be added into your schedule.
-* `YOUR_NUSMODS_URL` is obtained by clicking on the "Share/Sync" timetable icon at the NUSMods website.
+You can import your NUSMods timetable data into your task list by providing the URL to your NUSMods timetable share
+link. Follow these steps and try importing your timetable:
```

Comment:
You may want to consider telling the reader exactly where the task list is by providing a diagram of the GUI and highlighting `Upcoming tasks` or you can just mention that the task list is displayed under `Upcoming tasks` in the description :)